### ISSUE TITLE
Incorrect handling of inline comments

### ISSUE THREAD

**Issue by savchenko** [LudoPinelli/comment-box.nvim#34]
### Steps to reproduce

1. Create a Lua file:
```lua
    { -- Something begins here...
```
2. Execute `<Cmd>CBllline<CR>` while on the line.

### Desired result
```lua
    { -- Something begins here... ──────────────────────────────────────────────
```

### Actual result
```lua
-- ── { -- Something begins here... ──────────────────────────────────────────────
```

**Comment by LudoPinelli** [comment]
Yes, the plugin doesn't support inline comments at the moment. I have to mention it in the README.

It's seems kind of tricky to do, but I guess it could be a nice thing to have. I add it to the TODO list. Sorry for the disappointment and thanks for the suggestion!

**Comment by LudoPinelli** [comment]
> No disappointment at all. Would a naive check suffice at first? In pseudocode:
> 
> ```
> comment_pattern = regex_comment_pattern
> comment_pos = line.index(comment_pattern)
> 
> if comment_pos then
>     apply_comment_box.from(comment_pos)
> end
> ```

Probably, but at the moment, the code only knows the number of the line where the cursor is (or the first and last line selected in visual mode), not the position of the cursor in the line, so I need to implement that.
